src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AttitudeIndicator.js

```diff
@@ -1594,6 +1607,9 @@ class Jet_PFD_AttitudeIndicator extends HTMLElement {
                     this.radioDecisionHeight.textContent = fastToFixed(val, 0);
                 }
                 break;
+            case "half_bank-active":
+                this.cj4_HalfBankActive = parseFloat(newValue) > 15;
```

Comment:
Reverse this if you want to invert the arc display with the current state of the systems

Reply:
i already noticed. will do thanks!